cypress/integration/folder_navigation.js

```diff
@@ -1,3 +1,13 @@
+const click_menu_button = (name) => {
```

Comment:
Avoiding generic command for now because this doesn't seem to be "stable" enough. Relies on the dropdown not already being open.